content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -125,199 +125,195 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
-current-context: lithe-cocoa-92103_kubernetes
-kind: Config
-preferences: {}
-users:
-- name: lithe-cocoa-92103_kubernetes
-  user:
-    client-certificate-data: REDACTED
-    client-key-data: REDACTED
-    token: 65rZW78y8HbwXXtSXuUw9DbP4FLjHi4b
-- name: lithe-cocoa-92103_kubernetes-basic-auth
-  user:
-    password: h5M0FtUUIflBSdI7
-    username: admin
-```
+    ```shell
+    kubectl config view
+    ```
+    ```yaml
+    apiVersion: v1
+    clusters:
+     cluster:
+        certificate-authority-data: REDACTED
+        server: https://130.211.122.180
+      name: lithe-cocoa-92103_kubernetes
+    contexts:
+     context:
+        cluster: lithe-cocoa-92103_kubernetes
+        user: lithe-cocoa-92103_kubernetes
+      name: lithe-cocoa-92103_kubernetes
+    current-context: lithe-cocoa-92103_kubernetes
+    kind: Config
+    preferences: {}
+    users:
+     name: lithe-cocoa-92103_kubernetes
+      user:
+        client-certificate-data: REDACTED
+        client-key-data: REDACTED
+        token: 65rZW78y8HbwXXtSXuUw9DbP4FLjHi4b
+     name: lithe-cocoa-92103_kubernetes-basic-auth
+      user:
+        password: h5M0FtUUIflBSdI7
+        username: admin
+    ```
 
-```shell
-kubectl config current-context
-```
-```
-lithe-cocoa-92103_kubernetes
-```
+    ```shell
+    kubectl config current-context
+    ```
+    ```
+    lithe-cocoa-92103_kubernetes
+    ```
 
-The next step is to define a context for the kubectl client to work in each namespace. The values of "cluster" and "user" fields are copied from the current context.
+    The next step is to define a context for the kubectl client to work in each namespace. The values of "cluster" and "user" fields are copied from the current context.
 
-```shell
-kubectl config set-context dev --namespace=development --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
-kubectl config set-context prod --namespace=production --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
-```
+    ```shell
+    kubectl config set-context dev --namespace=development --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
+    kubectl config set-context prod --namespace=production --cluster=lithe-cocoa-92103_kubernetes --user=lithe-cocoa-92103_kubernetes
+    ```
 
-The above commands provided two request contexts you can alternate against depending on what namespace you
-wish to work against.
+    The above commands provided two request contexts you can alternate against depending on what namespace you
+    wish to work against.
 
-Let's switch to operate in the `development` namespace.
+    Let's switch to operate in the `development` namespace.
 
-```shell
-kubectl config use-context dev
-```
+    ```shell
+    kubectl config use-context dev
+    ```
 
-You can verify your current context by doing the following:
+    You can verify your current context by doing the following:
 
-```shell
-kubectl config current-context
-dev
-```
+    ```shell
```

Comment:
This isn't quite right (it mixes the command with its output), _but_ the fix should be a separate PR.